Various Fixes and DevEx improvements.

wkpatrick opened the pull request:
**I don't expect all of this PR to make it in, this is mostly just a collection of fixes and developer experience improvements.**

Fixes
- aviansie got renamed.
- Add try/catch statements around the builder loops. This way a bad article only affects itself, and not everything after it.
- Remove bare exception I added in wiki_page_text.py

Developer Experience Improvements
- Switch to using [uv](https://github.com/astral-sh/uv) for fetching deps and handling the venvs
- Add [mise](https://github.com/jdx/mise) for handling python installation and task running. It directly integrates with uv so it automatically switches to the venv when you enter the folder. This previously would also include a jdk installation, but this is no longer needed.
- Add a group of mise tasks that roughly map with the old update scripts. Easier to run and edit, and importantly will notify you of errors when one of the tasks errors out, instead of moving on to the next one (I think this is responsible for a bunch of new content not being added lately, especially with leagues and the temp items being around.
- Use a python script to download the latest pre-dumped caches from abex's github and extract it. This means we no longer need jdk/runlite in the project builder.

Let me know if you only want parts of this merged in, and I can make a new PR with just the changes you want. At least for me, ill be keeping all of it around.

wkpatrick committed:
Add starting mise file

wkpatrick committed:
Switch to uv, add trys to builder, fix bare exception in wiki text parser, add more mise tasks

wkpatrick committed:
Continue processing after errors in builders.

wkpatrick committed:
Add python script to download latest cache. Add update task to do everything.

wkpatrick committed:
Fix spacing in item builder

OSRSKoeppy commented:
@0xNeffarion I was wondering if you could take a look at @wkpatrick PR? 

Just was browsing through and I know we were using the data you are collecting for our runelite plugin but a handful of NPCs seem to be missing what drops they can give.

wkpatrick committed:
Fix build_item from stopping when it can't populate from the Wiki.
Properly set blank icon base64s.
Remove JDK from mise
Add Package task
Add a few different death versions.

DayV-git commented on data/schemas/schema-monsters.json:
```diff
@@ -134,7 +134,7 @@
       "abyssal demons",
       "adamant dragons",
       "ankou",
-      "aviansies",
```
Should add "crab" to this list as they have added that slayer category now.
